api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -142,8 +142,16 @@ message Bootstrap {
   }];
 
   // Optional watchdog configuration.
+  // This is for a single watchdog configuration for the entire system.
+  // For finer granularity watchdogs use *multi_watchdog*.
   Watchdog watchdog = 8;
```

Comment:
Should this be deprecated?

Reply:
Yeah, I'd be keen to not overlap mechanism long-term, so let's deprecate.